app/services/notifications/new_follower/follow_data.rb

```diff
@@ -12,7 +12,7 @@ class DataError < RuntimeError; end
       validates :followable_type, inclusion: { in: %w[User Organization] }
       validates :follower_id, numericality: { only_integer: true }
 
-      def initialize(*args, **kwargs)
+      def initialize(attributes)
```

Comment:
Keeping the argument pattern matching the[ method it's inherited(?) from](https://api.rubyonrails.org/classes/ActiveModel/Model.html).